articles/modules/ROOT/pages/what-is-pattern-comprehension-and-why-should-i-care.adoc

```diff
@@ -0,0 +1,78 @@
+= What is pattern comprehension and why should I care?
```

Comment:
While the title is catchy, it seems a bit more flippant than we probably need.

Maybe "and when should I use it" or "and why is it useful" or something similar.

If changing the title, make sure to change the slug and file name to match.